src/components/entity/list.vue

```diff
@@ -113,28 +119,51 @@ export default {
       })
     });
 
-    return { dataset, odataEntities, conflict };
+    const { request } = useRequest();
```

Comment:
Using `odataEntities.request()` makes some sense, as the DELETE request will have an effect on `odataEntities`: if the request is successful, then `odataEntities.countDeletion()` will be called. That said, it stretches the `requestData` model, because `requestData` resources are designed to have only a single request in progress at a time. Right now, if a request is sent while another request is in progress for the same resource, then the previous request is canceled. We could think about changing that in the future if it comes up again. For now, I think the best bet is to continue to use the `useRequest()` composable.